internal/exec/terraform.go

```diff
@@ -231,6 +231,14 @@ func ExecuteTerraform(info schema.ConfigAndStacksInfo) error {
 		}
 	}
 
+	// Check for any Terraform environment variables that might conflict with Atmos
+	for _, envVar := range os.Environ() {
+		if strings.HasPrefix(envVar, "TF_") {
+			varName := strings.Split(envVar, "=")[0]
```

Comment:
Not technically required, but for correctness.
```suggestion
			varName := strings.Split(envVar, "=", 2)[0]
```

Reply:
Are you sure?
```
internal/exec/terraform.go:237:45: too many arguments in call to strings.Split
	have (string, string, number)
	want (string, string)
make: *** [Makefile:31: build-default] Error 1
```

https://pkg.go.dev/strings#Split

Reply:
I think you mean `SplitN`. I will replace Split with SplitN